tangelo/linq/translator/translate_qiskit.py

```diff
@@ -65,27 +65,39 @@ def get_qiskit_gates():
 
 
 def translate_qiskit(source_circuit):
-    """Take in an abstract circuit, return an equivalent qiskit QuantumCircuit
-    instance
+    """Take in a Circuit, return an equivalent qiskit.QuantumCircuit
 
     Args:
-        source_circuit: quantum circuit in the abstract format.
+        source_circuit (Circuit): quantum circuit in the abstract format.
```

Comment:
Maybe we should stop using "abstract" and simply say "Tangelo" format everywhere, I think its clearer.
You can just focus on changing what you see in your PR, dont worry about the other instances :)